theme/templates/resource-landing-page/resource-header.html

```diff
@@ -108,28 +105,31 @@
                 </tr>
                 <tr>
                     <th>Created:</th>
-                    <td>{{ cm.created|date:"M d, Y" }} at {{ cm.created|time }}</td>
+                    <td><span class="local-datetime" data-dt="{{ cm.created|date:'c' }}"></span></td>
                 </tr>
                 <tr>
                     <th>Last updated:</th>
                     <td>
-                        {{ cm.last_updated|date:"M d, Y" }} at {{ cm.last_updated|time }}
+                        <span class="local-datetime" data-dt="{{ cm.last_updated|date:'c' }}"></span>
                         {% if cm.raccess.published and cm.last_updated > cm|published_date and cm.last_updated != cm|published_date %}
-                            <span>(Metadata update)</span>
+                        <span>(Metadata update)</span>
                         {% endif %}
                     </td>
                 </tr>
-                
+
                 {% if cm.raccess.published %}
                 <tr>
                     <th>Published date:</th>
-                    <td> {{ cm|published_date|date:"M d, Y" }} at {{ cm|published_date|time }}</td>
+
+                    <td>
+                        <span class="local-datetime" data-dt="{{ cm.published_date|date:'c' }}"></span>
+                    </td>
                 </tr>
                 <tr>
                     <th>DOI:</th>
                     <td>10.4211/hs.{{ cm.short_id }}</td>
                 </tr>
-                {% endif %}               
+                {% endif %}
```

Comment:
your edits lines 132-155 appear to all be whitespace/indentation. Was this intentional?
These changes make it difficult to review your PR.